xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/export_modal.vm

```diff
@@ -4,70 +4,82 @@
-      <h4 class="panel-title">
-        <a role="button" data-toggle="collapse" data-parent="#exportModelAccordion" href="#exportModelOfficeCollapse" aria-expanded="true" aria-controls="exportModelOfficeCollapse">
-          $escapetool.xml($services.localization.render('core.exporter.headings.officeFormats'))
-        </a>
-      </h4>
-    </div>
-    <div id="exportModelOfficeCollapse" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="exportModalHeadingOffice">
-      <div class="panel-body">
-        <a href="$doc.getURL('view', "xpage=pdfoptions&amp;qs=$!escapetool.url($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.pdf')</a>
-        #if("$!{services.officemanager.serverState}" == 'Connected')
-          <a href="$doc.getURL('export', "format=odt&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.odt')</a>
-          <a href="$doc.getURL('export', "format=rtf&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.rtf')</a>
-        #end
-        #foreach ($exportExtension in $exportExtensions)
-          #if ($exportExtension.parameters.category == 'office')
-            $services.rendering.render($exportExtension.execute(), 'html/5.0')
-          #end
+  <div class="panel-heading" role="tab" id="exportModalHeadingOffice">
+    <h4 class="panel-title">
+      <a role="button" data-toggle="collapse" data-parent="#exportModelAccordion" href="#exportModelOfficeCollapse"
+         aria-expanded="true" aria-controls="exportModelOfficeCollapse">
+        $escapetool.xml($services.localization.render('core.exporter.headings.officeFormats'))
+      </a>
+    </h4>
+  </div>
+  <div id="exportModelOfficeCollapse" class="panel-collapse collapse in" role="tabpanel"
+       aria-labelledby="exportModalHeadingOffice">
+    <div class="panel-body">
+      <a href="$doc.getURL('view', "xpage=pdfoptions&amp;qs=$!escapetool.url($request.getQueryString())")"
+         class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.pdf')</a>
+      #if("$!{services.officemanager.serverState}" == 'Connected')
+        <a href="$doc.getURL('export', "format=odt&amp;$!escapetool.html($request.getQueryString())")"
+           class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.odt')</a>
+        <a href="$doc.getURL('export', "format=rtf&amp;$!escapetool.html($request.getQueryString())")"
+           class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.rtf')</a>
+      #end
+      #foreach ($exportExtension in $exportExtensions)
+        #if ($exportExtension.parameters.category == 'office')
+          $services.rendering.render($exportExtension.execute(), 'html/5.0')
```

Comment:
Same here. Couldn't you fix the long lines without affecting the indentation of the other lines? It also makes the diff harder to read.